internal/impl/testdata/shellrc/basic/hook

```diff
@@ -1 +1 @@
-echo "Hello from a devbox shell hook!"
+/path/to/hooksfile/hooks.sh
```

Comment:
why is this file this? i dont get it

Reply:
it contains a (fake) path-to-the-hooks file.

I could: rename the file from `hooks.sh` to `path-to-hooks-file.txt`. 
Not sure how else to capture the two scenarios of this hooks file being missing or not. 🤔 is it always present?

Reply:
ah, okay, we always write this.

I will:
(a) delete this `shellrc/nohook` test case
(b) expose a function for the hooksFilePath and refer to that in the testcase logic.